lib/jazzy/config.rb

```diff
@@ -400,6 +401,12 @@ def print_attr_description(attr)
       end
     end
 
+    class PathnameArray < Array
```

Comment:
No, it’s that we’re applying logic to _all_ the configured attributed in one fell swoop. Most of them are always single paths, but one of is always an array.